### DISCUSSION TITLE
예제로 제공해 주신 ScreenShot API 로 찍으면 이상하게 나옵니다.

### DISCUSSION THREAD

**Discussion by peterkangdev** [naverz/zepeto-studio-kor#684] [Issue Report (Legacy)]
작은 체형의 아바타로 사진을 찍으면 이상하게 나옵니다. 
혹시 아시는 버그 인가요?

**[정상일떄]**
![정상아바타](https://user-images.githubusercontent.com/30133892/165984724-25b7dc96-c846-4d69-9d74-a3ab00405a64.jpeg)

**[비정상]**
![비정상 아바타](https://user-images.githubusercontent.com/30133892/165984769-a16e2709-e67d-4f51-a264-05788cabbb38.jpeg)

**Comment by pumfit** [comment]
아래 내용과 비슷 한 것 같아서 코드를 최신 버전으로 확인해보시면 좋을 것 같습니다!
https://github.com/naverz/zepeto-studio-kor/discussions/452#discussioncomment-2411490
